typed-builder-macro/src/field_info.rs

```diff
@@ -238,6 +239,10 @@ impl ApplyMeta for FieldBuilderAttr<'_> {
                 Ok(())
             }
             "setter" => self.setter.apply_sub_attr(expr),
+            "mutable_during_default_resolution" => {
+                self.mutable_during_default_resolution = true;
```

Comment:
Please add this line above:
```rust
expr.flag()?
```
To enforce that `mutable_during_default_resolution` is used correctly.

Even better - if you change the field type to `Span<Option>` you could use this:
```rust
"mutable_during_default_resolution" => expr.apply_flag_to_field(
    &mut self.mutable_during_default_resolution,
    "made mutable during default resolution",
),
```

Which would allow setting `mutable_during_default_resolution` in field defaults and removing it with `!mutable_during_default_resolution` for individual fields. The span is needed for better error reporting - and if we add it to the `mut` we'll get better "unused `mut`" warnings.